### PR TITLE
Implement Twilio outbound connector

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

Implement Twilio outbound connector

## Related issues

https://github.com/camunda/product-hub/issues/1138

closes #
https://github.com/camunda/connectors-bundle/issues/402

### CODE REVIEW

File: connectors/twilio/element-templates/twilio-connector.json [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+      },
+      "condition": {
+        "property": "authType",
+        "oneOf": [
+          "accountSidAuthToken"
+        ]
+      }
+    },
+    {
+      "label": "Auth token",
```
Comment (igpetrov) [inline connectors/twilio/element-templates/twilio-connector.json:224]: 1. Let's not use jargon `Auth` => `Authentication` etc.
2. Let's maintain the same style of labels everywhere. If you write here `Authentication token` then the next one has to be `Message text` and not `...Text` with capitalized first letter.

```suggestion
      "label": "Authentication token",
```

**Review by igpetrov — CHANGES_REQUESTED** [review]
LGTM, some changes requested.

File: connectors/twilio/element-templates/twilio-connector.json [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authType"
+      }
+    },
+    {
+      "label": "Account SID",
+      "description": "Your Twilio account SID. For more information, see <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio/\" target=\"_blank\">Camunda documentation</a>",
```
Comment (bastiankoerber) [inline connectors/twilio/element-templates/twilio-connector.json:103]: keep it consistent. Remove for more information and just say: see <link>

File: connectors/twilio/element-templates/twilio-connector.json [hunk 3]
```diff
@@ -0,0 +1,548 @@
...
+      "condition": {
+        "property": "authType",
+        "oneOf": [
+          "accountSidAuthToken"
+        ]
+      }
+    },
+    {
+      "label": "Authentication token",
+      "description": "Your Twilio auth token. For more information, see <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio/\" target=\"_blank\">Camunda documentation</a>",
```
Comment (bastiankoerber) [inline connectors/twilio/element-templates/twilio-connector.json:225]: remove for more information

File: connectors/twilio/element-templates/twilio-connector.json [hunk 4]
```diff
@@ -0,0 +1,548 @@
...
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authType",
+        "equals": "accountSidAuthToken"
+      }
+    },
+    {
+      "label": "Message SID",
+      "description": "The message SID to retrieve information about",
```
Comment (bastiankoerber) [inline connectors/twilio/element-templates/twilio-connector.json:243]: replace with: Obtain information regarding the message with SID

File: connectors/twilio/element-templates/twilio-connector.json [hunk 5]
```diff
@@ -0,0 +1,548 @@
...
+        "property": "operationType",
+        "oneOf": [
+          "getMessage"
+        ]
+      },
+      "optional": false
+    },
+    {
+      "label": "Message Text",
+      "description": "The text of the message to be sent",
```
Comment (bastiankoerber) [inline connectors/twilio/element-templates/twilio-connector.json:264]: replace with: The content of the message that will be sent

File: connectors/twilio/element-templates/twilio-connector.json [hunk 6]
```diff
@@ -0,0 +1,548 @@
...
+        "property": "operationType",
+        "oneOf": [
+          "sendSms"
+        ]
+      },
+      "optional": false
+    },
+    {
+      "label": "From Number",
+      "description": "The Twilio phone number to send the message from",
```
Comment (bastiankoerber) [inline connectors/twilio/element-templates/twilio-connector.json:285]: replace with: The originating Twilio phone number used for sending the message

**Review by bastiankoerber — COMMENTED** [review]
looks good, just some minor changes

**Review by igpetrov — APPROVED** [review]
LGTM, thank you.

**Pull request merged**